tests/unit/ember-sinon-qunit-test.js

```diff
@@ -56,3 +56,15 @@ test('sinon sandbox cleans up after itself', function (assert) {
   assert.ok(!spy.called, 'spy has no registered calls');
   assert.ok(!stub.called, 'stub has no registered calls');
 });
+
+test('does not fake timers by default', function (assert) {
+  assert.notEqual(Date.now(), 0, 'Date.now() is not reset');
+});
+
+test('does fake timers on request', { useFakeTimers: true }, function (assert) {
```

Comment:
the problem with this is that setting `sinon.config.fakeTimers` to `true` in a `beforeEach` block will not reset it again afterwards so you will need an `afterEach` block there too and then you may as well just use the fake timers directly without the sandbox.